pkg/agent/openflow/client.go

```diff
@@ -282,6 +286,12 @@ func (c *client) InstallDefaultTunnelFlows(tunnelOFPort uint32) error {
 	return nil
 }
 
+func (c *client) InstallBridgeUplinkFlows(uplinkPort uint32, bridgeLocalPort uint32) error {
+	flows := c.hostBridgeUplinkFlows(uplinkPort, bridgeLocalPort, cookie.Default)
+	c.hostNetworkingFlows = flows
```

Comment:
Please invoke `c.flowOperations.AddAll` first, if there is no error returned, update `c.hostNetworkingFlows`. This order could help keep the memory cache is the same as the flows on OVS.

Reply:
Done